src/style.css

```diff
@@ -0,0 +1,90 @@
+body {
+  margin: 0;
+  font-family: Arial, serif;
+  font-size: 100px;
+}
+
+.blocks__container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  flex-basis: 300px;
+  flex-shrink: 0;
+  flex-grow: 1;
+  color: white;
+}
+
+@media screen and (min-width: 600px) and (max-width: 899px) {
+  .block__one,
+  .block__two {
+    order: -1;
+  }
```

Comment:
If you set order manually, don't give same order value to several items, use unique numbers.